docs/source/index.rst

```diff
@@ -69,13 +71,6 @@ and run a simple client/server example:
     ------------------------------
 
 
-Documentation
```

Comment:
why removed? let's keep the link to PDF and HTML, sometimes it's more convenient

Reply:
I moved it to the head of API section.

Reply:
i wonder if we can put it to a more obvious place in the main page?

Reply:
IMHO - the original location was bad. I think placing this at the top of API section is a good place. People that care about API will click on the menu item ( on the left side) and the first thing they will see is the reference to full spec.

Reply:
okay